CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Do not attempt to auto-correct mass assignment or optional assignment in `Rails/RelativeDateConstant`. ([@rrosenblum][])
 * Fix auto-correction of `Style/WordArray` and `Style/SymbolArray` when all elements are on separate lines and there is a trailing comment after the closing bracket. ([@rrosenblum][])
 * Fix an exception that occurs when auto-correcting `Layout/ClosingParenthesesIndentation` when there are no arguments. ([@rrosenblum][])
+* [#6689](https://github.com/rubocop-hq/rubocop/pull/6689): Support more complex argument patterns on `Rails/Validation` auto-correction. ([@r7kamura][])
```

Comment:
Oops 💦 I forgot one piece of information. RuboCop 0.63.1 has been released. Please rebase with the latest master.

Reply:
Oh, I finally understand now why you mentioned about it.

Reply:
Thanks!